herbie/info.rkt

```diff
@@ -0,0 +1,11 @@
+#lang info
+(define collection "herbie")
+(define compile-omit-paths '("test" "reports/bash-pred-test.rkt" "herbie-web" "util.rkt"))
+(define raco-commands '(("herbie" (submod herbie/interface/inout main) "improve floating point expression accuracy" 75)))
+(define deps '("base"
+               "math-lib"
+               "plot-lib"
+               "profile-lib"
+               "rackunit-lib"
+               "srfi-lite-lib"
+               "web-server-lib"))
```

Comment:
I'm not sure where it's used, but I know it is. These dependencies were automatically discovered by `raco`, and its smart enough to know to exclude dependencies needed by things in `compile-omit-paths`.